File: src/vehicle/rental/system/vehicleType/Car.java
package vehicle.rental.system.vehicleType;

import vehicle.rental.system.rental.Vehicle;

import java.math.BigDecimal;
import java.math.RoundingMode;

public class Car extends Vehicle {

    private final int safetyRating;

    public Car(String brand, String model, BigDecimal value, int safetyRating) {
        super(brand, model, value, "car");
        this.safetyRating = safetyRating;
    }

    public int getSafetyRating(){return safetyRating;}

    public void displayInfo(){
        System.out.println("\nA car that is valued at $" + getValue().setScale(2, RoundingMode.DOWN) + " and has a security rating of " + safetyRating + "\n");
    }
}

File: src/VehicleType/Car.cs
using VehicleRentalSystem.Rental;

namespace VehicleRentalSystem.VehicleType;

public class Car : Vehicle
{
    public int SafetyRating { get; }

    public Car(string brand, string model, decimal value, int safetyRating)
        : base(brand, model, value, "car")
    {
        SafetyRating = safetyRating;
    }

    public void DisplayInfo()
    {
        Console.WriteLine($"\nA car that is valued at ${Value:F2} and has a security rating of {SafetyRating}\n");
    }
}
